src/components/UnovaSVG.html

```diff
@@ -846,4 +831,87 @@
         })
     %>
 
+    <!--Plasma Frigate 1, Castelia City-->
+    <%- include('templates/mapOverlay',
+        { name: "A very normal frigate"
+        , x: 39, y: 52
+        , width: 4, height: 9
+        , databind: `attr: { class: App.game.statistics.gymsDefeated[GameConstants.getGymIndex('Virbank City')]() >= 1 },
```

Comment:
This class-attribute does not do anything, so let's remove it

Reply:
Removed class-attribute, removed Opelucid City MoveToDungeon.
Not gonna remove Ghetsis from Giant Chasm after first clear. We don't remove story bosses from other dungeons.